base/interactiveutil.jl

```diff
@@ -667,7 +667,7 @@ Print information about exported global variables in a module, optionally restri
 
 The memory consumption estimate is an approximate lower bound on the size of the internal structure of the object.
 """
-function whos(io::IO=STDOUT, m::Module=current_module(), pattern::Regex=r"")
+function whos(io::IO=STDOUT, m::Module=current_module(), pattern::Regex=r"^(?!.*(Base|Main|Core)).*$")
```

Comment:
this would also filter names like `StatsBase`. Maybe it should just skip calling `summarysize` on `value in (Base, Main, Core)`?